denonavr/const.py

```diff
@@ -375,6 +408,17 @@
     command_set_all_zone_stereo=COMMAND_SET_ZST_URL,
     command_pause=COMMAND_PAUSE,
     command_play=COMMAND_PLAY,
+    command_cusor_up=COMMAND_CURSOR_UP,
+    command_cusor_down=COMMAND_CURSOR_DOWN,
+    command_cusor_left=COMMAND_CURSOR_LEFT,
+    command_cusor_right=COMMAND_CURSOR_RIGHT,
+    command_cusor_enter=COMMAND_CURSOR_ENTER,
+    command_back=COMMAND_BACK,
+    command_info=COMMAND_INFO,
+    command_options=COMMAND_OPTIONS,
+    command_setup_open=COMMAND_SETUP_OPEN,
+    command_setup_close=COMMAND_SETUP_CLOSE,
+    command_setup_query=COMMAND_SETUP_QUERY
```

Comment:
These lines should be added to `ZONE2_URLS` and `ZONE3_URLS` too.
This might be the reason why the tests are failing.